frontend/.vscode/launch.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
I think this happens when the drawing is not calibrated on the model I am looking at, but in that case, I get the "offSightCamera" icon which doesn't point to any of the closest edges. Also, moving around the viewpoint in the 3d model doesn't affect it at all

![image](https://github.com/user-attachments/assets/4594a652-24ce-4b1a-a905-e221adafbd29)

Reply:
This one in particular has the wrong hardcoded calibration. If the drawing has no calibration the camera doesnt appear.